discovery/README.md

```diff
@@ -10,8 +10,19 @@ Docker Swarm comes with multiple Discovery backends.
 
 ## Backends
 
+You use a hosted discovery service with Docker Swarm. The service
+maintains a list of IPs in your swam. There are several available
+services, such as `etcd`, `consul` and `zookeeper` depending on what
```

Comment:
@thaJeztah I see your point but really it is splitting hairs...I'd rather merge it then send Morgan back for such a fine grained change.  LGTM